### ISSUE TITLE
Strange...

### ISSUE THREAD

**Issue by nadavb** [mouradmourafiq/tensorflow-lstm-regression#26]
on lstm_sin, I change the training_steps from 10000 to 10, and the batch size from 100 to 5, and get the same results...

Are you sure the whole thing is really predicting?

**Comment by mouradmourafiq** [comment]
can you provide more information about your setup, tensorflow version, ...

**Comment by mouradmourafiq** [comment]
I just run the example with 10 steps and a batch size 5, and got this result:

![image](https://cloud.githubusercontent.com/assets/1261626/26310115/f26f713c-3eff-11e7-8cc7-0023e261ce81.png)

Obviously it did not learn yet to predict a correct sin function. And even if it's the case, the sin function is very simple to learn for the model. since it's monotonic, this would not work for more complex function or time series, try to do the same for x*sin and see for your self.

One last thing, if you ran the example before with 10000/100 and then changed the values, of course it will predict with the same accuracy because you basically just added a few steps to already trained model. The variables are saved after each execution, you need to remove the folder where the data is stored.

Closing this issue.

**Comment by nadavb** [comment]
You're right, the model was cached in the filesystem.
